Zend/zend_execute.c

```diff
@@ -1438,7 +1440,7 @@ static zend_never_inline void zend_pre_incdec_overloaded_property(zval *object,
 		}
 		Z_OBJ_HT(obj)->write_property(&obj, property, z, cache_slot);
 		OBJ_RELEASE(Z_OBJ(obj));
-		zval_ptr_dtor(z);
+		zval_ptr_dtor(zptr);
```

Comment:
Note: this zptr is an unrelated memory leak, which I discovered while writing the phpt test; it was leaking the reference. (subtest 14)